sass/directives/02_base_components/_modals.scss

```diff
@@ -31,3 +31,16 @@
     text-decoration: none;
   }
 }
+
+@mixin close-x-overflow {
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+  background-color: #333;
+  opacity: 1;
+  color: white;
+  line-height: 0;
+  font-size: 16px;
+  right: -10px;
```

Comment:
looking at the redlines doc, does the corner meet right in the middle of the circle? (i'm not entirely sure, you probably reviewed this with Susan before she left) If that's the case and if you use a variable for the height and width of the circle, you could use `-$feature-modal-controls-size / 2` here